pyproject.toml

```diff
@@ -54,6 +54,10 @@ runtime-http = [
     "sse-starlette>=1.6.1,<2",
 ]
 
+runtime-client = [
```

Comment:
heh, ug, it's so hard to slice-and-dice requirements. Technically, this _should_ only be needed for `protocol = "http"`. In code, it's probably required either way at this point, but that could be fixed with localized imports

Reply:
Yeah, they'd just have to be narly localized imports inside functions which is not great. I added a comment specifying this is only required for HTTP clients